specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -82,6 +82,61 @@
         }
       }
     },
+    "/managedidentitymodebackup": {
```

Comment:
POST operations should take the form of a verb. Maybe that's what "backup" means here (more like "back up" sans spaces), but this long endpoint name isn't very descriptive nor does it follow many of our recommended patterns. For example, shouldn't this build on the existing /backup endpoint? What exactly is the "Manage Identity Mode" adding here? Isn't that just an authentication schema? Can this just be a change to the input model for /backup?